### DISCUSSION TITLE
Consumir WebServices NF-e

### DISCUSSION THREAD

**Discussion by souzadeveloper** [balta-io/forum#375] [Tira Dúvidas] [answered]
Fala Balta blz? É possível consumir os WebServices da NF-e (Nota Fiscal Eletrônica) usando .net 6 ou 7 ou somente através de .Net Standard mesmo? Pelo que vi que o namespace System.Web.Services foi descontinuado e não consegui de forma alguma consumir um dos WebSerices das Sefaz usando .net 6.

Agradeço qualquer ajuda

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @souzadeveloper, como vai?

Acredito que seja possível sim:
https://medium.com/compendium/integrating-with-soap-web-services-in-net-core-adebfad173fb

Porém faz um bom tempo que não trabalho com SOAP/XML!

**Comment by souzadeveloper** [comment]
Bom dia Balta, realmente eu vi que só tem como trabalhar usando Soap como se fosse um WCF (igual ao do Artigo), porém a comunicação não funciona... Você saberia informar como seria a configuração dos Bindings quando é exigido um Certificado em requisiçoes HTTPS.

Obtive os erros abaixo:

1 - A solicitação HTTP está proibida com o esquema de autenticação de cliente 'Anonymous'
2 - Possible SOAP version mismatch

Obrigado.

**Reply by andrebaltieri** [reply]
@souzadeveloper essa parte vou ficar devendo!! Chama no Discord que te passo uns contatos que podem te ajudar.
https://balta.io/discord
